Backbone.Infrastructure/UserContext/AspNetCoreUserContext.cs

```diff
@@ -59,6 +59,17 @@ public string GetUserIdOrNull()
         return userId;
     }
 
+    public string GetUsername()
+    {
+        return GetUsernameOrNull() ?? throw new NotFoundException();
+    }
+
+    public string GetUsernameOrNull()
+    {
+        var username = _context.HttpContext.User.Identities.First().Name;
```

Comment:
1. First() throws an exception if there is no element.
2. Why don't you use the same way to retrieve the value as in the other methods? (`_context.HttpContext.User.FindFirstValue(USERNAME_CLAIM)`)?

Reply:
2. I considered it might be more interesting to use the typed attribute, but using the `name` claim works too.

Reply:
Hm, yeah. If it's actually the correct value, why not